example/lib/main.dart

```diff
@@ -62,78 +72,81 @@ class _DemoState extends State<Demo> {
       ),
       sidebar: Sidebar(
         minWidth: 200,
-        builder: (context, _) => ListView(
-          children: [
-            Container(
-              padding:
-                  const EdgeInsets.symmetric(horizontal: 16.0, vertical: 8.0),
-              child: GestureDetector(
-                onTap: () {
+        builder: (context, _) => FocusScope(
```

Comment:
Why the `FocusScope`?

Reply:
It's leftover from my attempts to manage the widget using Focus